app/templates/customer/new_customer.html.j2

```diff
@@ -121,14 +74,15 @@
 
     ////////////////////////////////////////////////////////////////////////////////////////////////////
     // Make sure the entered email address is a vaild email format
+    // emailRegExp source: https://www.w3resource.com/javascript/form/email-validation.php
     ////////////////////////////////////////////////////////////////////////////////////////////////////
     function emailValidate() {
         const email = document.getElementById('email');
+
         const emailRegExp =
-        /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+        /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
This regex doesn't match the one above in the other file.  Also, a + sign needs to be allowed as I previously mentioned.

Also, since this code appears in more than one location, maybe we should take it out and put it somewhere central?  That way, when you update the code once, the code is the same everywhere it's used.

Reply:
I added the plus sign, I couldn't tell what other part didn't match though.

I agree, its on my todo list to create a Javascript directory under the Static directory and move all the scripts there to help avoid that duplication. Just been contemplating how to organize the JS files.

1.  Have all the scripts in one file, such as Main.js
2. Organized by "function", so an inputValidation.js, deleteButton.js, Modals.js, etc.
3. Have them by page, newCustomer.js, login.js, editCustomer.js

Reply:
This looks good.  They are all the same.